lib/assessment.js

```diff
@@ -29,7 +44,8 @@ var newArray = function newArray(length, defaultValue) {
 // invoking the argument defaultsFunction
 // defaultsFunction should be passed the index of the array element to be set
 var newArray2 = function newArray2(length, defaultsFunction) {
-
+  var array2 = newArray(length, defaultsFunction);
+  return array2;
```

Comment:
Not quite - the directions say that each element should have the value `defaultFunction(i)`. 
i.e.
`[defaultFunction(0), defaultFunction(1), ...]`
Given that, do you think you could rewrite your solution?